data/crac-file/crac-api/src/test/java/com/farao_community/farao/data/crac_api/InstantTest.java

```diff
@@ -22,17 +22,44 @@
 
     @Before
     public void setUp() {
-        instant = new Instant(15);
+        instant = new Instant("instant", 15);
     }
 
     @Test
     public void getDuration() {
-        assertEquals(15, instant.getDuration(), 0.1);
+        assertEquals(15, instant.getSeconds(), 0.1);
```

Comment:
You now compare ints and not doubles, right? You can remove the epsilon = 0.1, and just leave assertEquals(15, instant.getSeconds()), isn't it?